server/controllers/answers.js

```diff
@@ -23,7 +23,7 @@ const Question = require("../models/schema/posts");
   // Post an answer to a question
 module.exports.post_ans = async (req, res) => {
   let question_id = req.params.id;
-  console.log(question_id);
+  // console.log(question_id);
```

Comment:
Good job @Kalivan3, please remove/delete all the console.log instead of commenting them out.

Reply:
Okay. Let me work on that. I hope Elywin will do the same